pkg/cmd/cli/describe/describer.go

```diff
@@ -249,6 +250,10 @@ func describeBuildSpec(p buildapi.BuildSpec, out *tabwriter.Writer) {
 			formatString(out, "Revision Message", p.Revision.Git.Message)
 		}
 	}
+
+	if p.CompletionDeadlineSeconds != nil {
+		formatString(out, "Completion Deadline Seconds", time.Duration(*p.CompletionDeadlineSeconds)*time.Second)
```

Comment:
So far our descriptions are quite similar to var names, since it's easier to know what you're talking about. Must complete would make it less obvious.